lib/routes/termsAndConditionsRoutes.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+const nmd = require('nano-markdown')
+
+const router = express.Router()
+
+const configManager = require('../new-config-manager')
+const plugins = require('../plugins')
+
+function createTerms (terms) {
+  if (!terms.active || !terms.text) return null
+
+  return {
+    active: terms.active,
+    title: terms.title,
+    text: nmd(terms.text),
+    accept: terms.acceptButtonText,
+    cancel: terms.cancelButtonText
+  }
+}
```

Comment:
This function is equal to other one i commented on, right? Maybe you can reuse it, or else change to ternary operator